dm/dm/master/scheduler/scheduler.go

```diff
@@ -330,7 +330,7 @@ func (s *Scheduler) CloseAllWorkers() {
 
 // AddSourceCfg adds the upstream source config to the cluster, and try to bound source to worker
 // NOTE: please verify the config before call this.
-func (s *Scheduler) AddSourceCfg(cfg *config.SourceConfig) error {
+func (s *Scheduler) AddSourceCfg(cfg *config.SourceConfig, tryBound bool) error {
```

Comment:
I think the `tryBound` parameter is not needed. You can use `AddSourceCfgWithWorker` when OpenAPI specifies a worker name, and `AddSourceCfg` when not. Also, TransferSource is not needed to be changed

Reply:
yes, i can use `AddSourceCfgWithWorker` instead revert these changes in https://github.com/pingcap/tiflow/pull/4510/commits/38d4ddab34f43c26e540f02b55817991d62b896e and https://github.com/pingcap/tiflow/pull/4510/commits/b8d573516f216d7daeae3073e0d5648daf30cec3